package.json

```diff
@@ -6,6 +6,7 @@
   "author": "George MacKerron",
   "license": "MIT",
   "dependencies": {
+    "@prisma/adapter-neon": "^6.2.1",
```

Comment:
Is this used on code or just tests? If it's just tests, let's move it to `devDependencies`

Reply:
Because of the way we currently have a separate `package.json` for the `npm` folder, this doesn't matter in practice. But you're right, and I have moved it.

Again, this is all likely to change as part of the upcoming refactor.